sql/python/sqlflow/template/test_sqlite3_db.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
```

Comment:
Would you mind enable the test in our CI? This can be done at `scripts/test.sh`.

If you wanna test against Hive, please refer to modify `scripts/test_hive.sh`.